### ISSUE TITLE
may a name mistake in beta version (utilities.py)

### ISSUE THREAD

**Issue by delta-me13** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#334]
hello Author.
I think I have found a name error in the test version v0.3.0 code, this error is related to the file utilities.py. Here is the error report file
```python
File "/home/firefly/ai/stable-diffusion-webui/extensions/Stable-Diffusion-WebUI-TensorRT-0.3.0/utilities.py", line 243, in build
    _p.add(namFe, min=dims[0], opt=dims[1], max=dims[2])
NameError: name 'namFe' is not defined. Did you mean: 'name'?
```
When I corrected this error, the program ran correctly. So I think the reason for the error is this.
This is the first time I've raised lusse on github, so please forgive me if I've done some wrong!

**Comment by GrogPower** [comment]
Yes. In file "utilities.py" string "_p.add(namFe, min=dims[0], opt=dims[1], max=dims[2])" must be changed on "_p.add(name, min=dims[0], opt=dims[1], max=dims[2])". Right version string from release v0.2.1 "utilities.py".
